src/config/dynamic-theme-fixes.config

```diff
@@ -12,6 +12,10 @@ CSS
 ::placeholder {
     opacity: 0.5 !important;
 }
+.CodeMirror-selected {
+    color: &{selectionForeground} !important;
```

Comment:
Change of plans.

`color: var(--darkreader-selection-fg) !important;`
`background-color: var(--darkreader-selection-bg) !important;`

Variables are easier to use.
https://github.com/darkreader/darkreader/pull/2767#issuecomment-656637478